staging/copy.sh

```diff
@@ -120,6 +120,21 @@ find "${MAIN_REPO}/pkg/version" -maxdepth 1 -type f | xargs -I{} cp {} "${CLIENT
 mkcp "pkg/client/clientset_generated/${CLIENTSET}" "pkg/client/clientset_generated"
 mkcp "pkg/client/informers/informers_generated/externalversions" "pkg/client/informers/informers_generated"
 
+echo "copy repos in staging/ to temp dir"
+kube::util::ensure-temp-dir
```

Comment:
This conflict with the cleanup function above: there is no chain of trap functions, only one. Can we switch completely to `KUBE_TEMP`? No need for our own tmp dir logic.